pkg/liveshare/client.go

```diff
@@ -136,41 +135,3 @@ type joinWorkspaceArgs struct {
 type joinWorkspaceResult struct {
 	SessionNumber int `json:"sessionNumber"`
 }
-
-// A channelID is an identifier for an exposed port on a remote
-// container that may be used to open an SSH channel to it.
-type channelID struct {
-	name, condition string
-}
-
-func (s *Session) openStreamingChannel(ctx context.Context, id channelID) (ssh.Channel, error) {
```

Comment:
I moved the definitions of these methods on `Session` into the `session.go` file.